src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,62 @@
+.stopwatch {
+  display: flex;
+  height: 100vh;
+
+  &__inner {
+    position: relative;
+    border: 1px solid black;
+    background-color: antiquewhite;
+    border-radius: 50%;
+    height: $stopwatch-size;
+    width: $stopwatch-size;
+    margin: auto;
+    display: flex;
+  }
+
+  &__center {
+    height: $item-size;
+    width: $item-size;
+    background-color: #555;
+    border-radius: 50%;
```

Comment:
You can create a mixin for making circle

Reply:
agree, mixin looks good here